### PR TITLE
updated to configuration for the swarm to do more retires for unrespo…

### PR DESCRIPTION
**Pull request by challa**
swarm manager marks the docker node as unhealthy with-in 3 tries  with heart-beat interval of 20  seconds  if docker demon does not respond back with in that time. Swarm manager does not ping back the unhealthy nodes

During the service-fabrik deployment node update takes around 4 to 8 Minutes and some nodes remain unhealthy until deployment is finished in which swarm manger is restarted.

Increasing the swarm retries to 30 times i.e swarm will ping for 10 minutes before it marks a node as unhealthy

### CODE REVIEW

File: jobs/swarm_manager/spec [hunk 1]
```diff
@@ -46,6 +46,9 @@ properties:
   swarm_manager.heartbeat:
     description: "Time in second between each heartbeat"
     default: "20s"
+  swarm_manager.node-heartbeat-retries:
+    description: " No of retries swarm manager makes to check if docker nodes is healthy  again once its finds it un-healthy"
```
Comment (sauravmndl) [inline jobs/swarm_manager/spec]: " No of retries ..."
Extra space @beginning

File: jobs/swarm_manager/spec [hunk 2]
```diff
@@ -46,6 +46,9 @@ properties:
   swarm_manager.heartbeat:
     description: "Time in second between each heartbeat"
     default: "20s"
+  swarm_manager.node-heartbeat-retries:
```
Comment (sauravmndl) [inline jobs/swarm_manager/spec]: "node-heartbeat-retries" --> "node_heartbeat_retries" as other params are also following same.

**Comment by challa** [comment]
@sauravmndl  made changes as per the review comments, could you please check

File: jobs/swarm_manager/templates/bin/job_properties.sh.erb [hunk 3]
```diff
@@ -56,6 +56,9 @@ export SWARM_MANAGER_ADVERTISE="--advertise=<%= "#{p('swarm_manager.advertise',
 # Time in second between each heartbeat
 export SWARM_MANAGER_HEARTBEAT="--heartbeat=<%= p('swarm_manager.heartbeat') %>"
 
+# No of retries swarm manager makes to check if docker nodes is healthy again if it was un-healthy previously
+export SWARM_MANAGER_HEARTBEAT_RETRIES="--engine-failure-retry=<%= p('swarm_manager.node-heartbeat-retries') %>"
```
Comment (sauravmndl) [inline jobs/swarm_manager/templates/bin/job_properties.sh.erb]: node-heartbeat-retries......should it be "node_heartbeat_retries" ?

**Comment by challa** [comment]
incorporated the changes, please cross check again & merge

**Pull request merged**